src/Anonymization/Anonymizator.php

```diff
@@ -127,6 +126,7 @@ public function anonymize(
         }
 
         $plan = [];
+        $context = clone $this->defaultContext;
```

Comment:
Given taht `$context` is `readonly` in `AbstractAnonymizer`, why do you need to clone it here ?